src/main/java/me/creighton/encodedid/EncodedIdException.java

```diff
@@ -25,43 +25,48 @@ public EncodedIdException(String message, Throwable cause) {
    */
 
   public static void throwInvalidSeparator (char separator, String alphabet) throws EncodedIdException {
-    throw new EncodedIdException("Invalid Separator: " + separator + " for alphabet: " + alphabet);
+    throw new EncodedIdException("Exception!! Invalid Separator: " + separator + " for alphabet: " + alphabet);
```

Comment:
We don't typically prefix exception messages with Exception. Since it's a cross-cutting concern, the typical approach is to specify something like this in a logback.xml so this pattern can be applied once that applies to all exceptions that are logged.

Reply:
I removed all the prefixes in the exception messages. They were supluous.